solution/build.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+PWD=$(pwd)
+BUILD_ROOT=${PWD}/_build
+ISO_ROOT=${BUILD_ROOT}/root
+IMAGES_ROOT=${ISO_ROOT}/images
+
+PRODUCT_NAME=Zenko
+PRODUCT_LOWERNAME=zenko
+BUILD_TIMESTAMP=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+BUILD_HOST=$(hostname)
+
+VERSION_SHORT=$(git describe --abbrev=0)
+VERSION_FULL=${VERSION_SHORT}-dev
+GIT_REVISION=$(git describe --long --always --tags --dirty)
+ISO=${BUILD_ROOT}/${PRODUCT_LOWERNAME}-${VERSION_FULL}.iso
+
+DOCKER=docker
+DOCKER_OPTS=
+DOCKER_SOCKET=unix:///var/run/docker.sock
+HARDLINK=hardlink
+OPERATOR_SDK=operator-sdk
+OPERATOR_SDK_OPTS=
+SKOPEO=skopeo
+SKOPEO_OPTS="--override-os linux --insecure-policy"
+OPERATOR_TAG=$(grep /zenko-operator: deps.txt | awk -F ':' '{print $2}')
+
+# grab our dependencies from our deps.txt file as an array
+readarray -t DEP_IMAGES < deps.txt
+
+function clean()
+{
+	rm -rf ${BUILD_ROOT}
+}
+
+function mkdirs()
+{
+	mkdir -p ${ISO_ROOT}
+}
+
+function copy_yamls()
+{
+	sed "s/ZENKO_OPERATOR_TAG/${OPERATOR_TAG}/" config.yaml > ${ISO_ROOT}/config.yaml
+	cp -R -f operator/ ${ISO_ROOT}/operator
+	cp zenko-version-cr.yaml ${ISO_ROOT}/operator/zenko-version-cr.yaml
+}
+
+function download_dependency_yamls()
+{
+	OPERATOR_DIR=${ISO_ROOT}/operator
+	CRDS_DIR=${OPERATOR_DIR}/crds
+
+	wget --directory=${CRDS_DIR} https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yaml
+	wget -O ${CRDS_DIR}/prom-operator-bundle.yaml https://github.com/coreos/prometheus-operator/blob/v0.38.0/bundle.yaml
```

Comment:
The prometheus operator already runs as part of metalk8s (I think) so we don't need to install it.

The manifests listed here look good otherwise, it's missing only kubedb (that manages redis and mongodb). Kubedb's a bit trickier as it's not a clean set of yaml manifests but an installer script: https://raw.githubusercontent.com/kubedb/installer/89fab34cf2f5d9e0bcc3c2d5b0f0599f94ff0dca/deploy/kubedb.sh . It seems that the script can be "flattened" to get only the yaml files we need here, as well as maybe a `Job` resource to execute whatever commands we can't get away without.

Reply:
oh I seem to have missed the kubedb link Alex sent me yesterday, https://github.com/kubedb/installer/tree/v0.13.0-rc.0/deploy is what he sent. I can flatten that folder into a bundle like the others if it works. Or take a look at the shell script to see what I can pull. Still very new to this so not sure what is more appropriate.

Reply:
should go with @RachedBenMustapha suggestions of https://raw.githubusercontent.com/kubedb/installer/89fab34cf2f5d9e0bcc3c2d5b0f0599f94ff0dca/deploy/kubedb.sh  rather than the `v0.13.0-rc.0` version; the `v0.13.0-rc.0` version has some issues installing on kubernetes version 1.16+, which metalk8s seems to be running

Reply:
I strongly suggest looking inside the installer script and adapting the bundle for our needs:

- we don't need most of the versions from the catalog (actually none since zenko-operator has a PR in review to generate our own versions)
- we should be mindful that the namespace kubedb will be deployed in is not going to be its default (kube-system) so this may have an impact on actual solution "activation" in its own namespace by metalk8s.
- in the first iteration we could do without things like pod security policies, mutating webhooks, validating webhooks, ...

Alternatively kubedb has a chart here: https://github.com/kubedb/installer/tree/master/charts/kubedb that can be "templated" with helm to generate yaml which could be convenient (or not).

Reply:
@RachedBenMustapha Mind if I hold off on the kubedb work until the next PR? Get this one merged and work on the kubedb after. Try and keep the PR's relatively small.